GliaWidgets/Sources/CallVisualizer/ScreenSharing/View/ScreenSharingView.swift

```diff
@@ -99,9 +110,9 @@ private extension CallVisualizer.ScreenSharingView {
             for: messageLabel
         )
 
-        header.title = props.style.title
-        header.backButton.tap = props.back.execute
-        endScreenSharingButton.tap = props.endScreenSharing.execute
+        header.props.title = props.style.title
```

Comment:
Similar to previous comments I would suggest to avoid mutating props properties directly.